### PR TITLE
ELK stack upgrade procedure added

### PR DESCRIPTION
**Pull request by aremai**

### CODE REVIEW

File: source/ossec_elk_upgrade.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+3. **Important: Backup the Elasticsearch Index** everything in /var/lib/elasticsearch/ossec/*
+4. stop the Elasticsearch service
+5. Update the Yum Repo with the latest Elasticsearch version. Current version recommended by WAZUH is Elasticsearch
+6. run “yum update elasticsearch”
+7. Replace the elasticsearch.yml file with the new. Check if the relevant lines from above are still there (cluster.name and node.name)
+8. Verify that Elasticsearch is running by issuing in your terminal:
+   curl -XGET localhost:9200
+   curl -XGET 'http://localhost:9200/_cluster/health?pretty=true'
+
+finally run: ~/ossec-wazuh/extensions/elasticsearch/ && curl -XPUT “http://localhost:9200/_template/ossec/” -d “@elastic-ossec-template.json”
```
Comment (netflash) [inline source/ossec_elk_upgrade.rst]: cd ~/ossec ?
please use \` sign to escape the code
and this doublequotes won't work in console

**Comment by jlruizmlg** [comment]
hi @netflash this pull request is not accepted yet, is under review

**Comment by netflash** [comment]
Hi @jlruizmlg . Yeah, I got it. Just some things that I spotted.

**Comment by jlruizmlg** [comment]
@netflash Thanks!!!

**Pull request closed without merging**